src/Engine/Animator/Animator.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include "../Engine/GraphicsHandler/I_GraphicsHandler.h"
+#include <vector>
+
+class Animator {
+public:
+    /**
+     * Creates a Sprite Animator object
+     */
+    Animator();
+    /**
+     * Uses the <code>SDL_Texture*</code> to create a <code>SDL_Rect</code> that represents the
+     * spritesheet.
+     */
+    int init(SDL_Texture* texture, int rows, int cols, int period);
+    /**
+     * Uses the <code>SDL_Texture*</code> to create a <code>SDL_Rect</code> that represents the
+     * spritesheet.
```

Comment:
I don't think this comment is accurate. the `init` function doesn't really create anything.